sbin/spark-daemon.sh

```diff
@@ -74,10 +75,13 @@ shift
 spark_rotate_log ()
 {
     log=$1;
-    num=5;
-    if [ -n "$2" ]; then
-	num=$2
+
+    if [[ ${SPARK_LOG_MAX_FILES} -gt 5 ]] 2>/dev/null; then
+      num=${SPARK_LOG_MAX_FILES}
+    else
+      num=5
```

Comment:
The change itself looks okay to me, but current users who use the `$2` argument might get surprised at this change a bit?

Reply:
not really. the `$2` is not provided via its only caller.
https://github.com/apache/spark/blob/d273a2bb0fac452a97f5670edd69d3e452e3e57e/sbin/spark-daemon.sh#L173-L174

The log max number is always 5.

Reply:
Oh, I see. I got it.